### ISSUE TITLE
feature: support Debian 12 bookworm

### ISSUE THREAD

**Issue by onitake** [microsoft/playwright#24028]
**Context:**
- GOOD Playwright Version: N/A
- BAD Playwright Version: 1.35.0
- Operating System: Debian bookworm
- Extra: The installer still worked on Debian bullseye

**Code Snippet**

```docker
FROM python:3.11-slim-bookworm
RUN pip install playwright==1.35.0
RUN playwright install firefox --with-deps
```

**Describe the bug**

Similar to the issues in #13530 , playwright fails to install on Debian bookworm:
```
E: Package 'ttf-unifont' has no installation candidate
E: Package 'xfonts-cyrillic' has no installation candidate
E: Package 'ttf-ubuntu-font-family' has no installation candidate
```

The workaround with the non-free repository doesn't help, because these packages have different issues:

* For `ttf-unifont`, please install `fonts-unifont` instead. The `ttf-` prefix has been dropped in Ubuntu 22.04 and Debian bookworm in favor of `fonts-`.
* For `ttf-ubuntu-font-family`, please use `fonts-ubuntu`. Same reason as above. Additionally, this package is only available from the non-free repository on Debian, so I think it should be dropped unless it's really needed.
* `xfonts-cyrillic` is not available in Debian stable/testing due to https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=999227 , but is this really needed? Bitmap fonts do not scale well on high-DPI screens. If Unifont doesn't cover the Cyrillic alphabet sufficiently and you really need a bitmap font, please use something like xfonts-cronyx-75dpi that is available both in Debian and Ubuntu. Otherwise, I'd simply drop this dependency.

**Comment by racam** [comment]
Same problem here. Fixed it by forcing bullseye `FROM python:3.10.12-slim-bullseye`

**Comment by mxschmitt** [comment]
Folding into https://github.com/microsoft/playwright/issues/23532
